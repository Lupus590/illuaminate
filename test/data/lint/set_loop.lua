-- config: (lint (only (var:set-loop)))
for x = 0, 10 do
  x = x + 1
end

for k in pairs(_G) do
  k = 0
end
