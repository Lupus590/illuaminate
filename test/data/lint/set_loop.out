set_loop.lua:[3:3-3:3]: Mutating loop variable "x". [var:set-loop]
   │
 3 │   x = x + 1
   │   ^
set_loop.lua:[7:3-7:3]: Mutating loop variable "k". [var:set-loop]
   │
 7 │   k = 0
   │   ^
No errors and 2 warnings
